plugins/node/opentelemetry-winston-transport/README.md

```diff
@@ -0,0 +1,68 @@
+# OpenTelemetry transport for winston
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+This module provides a Transport for [`winston`](https://www.npmjs.com/package/winston) module to send Winston logging to the OpenTelemetry Logging SDK. It may be loaded using the [`@opentelemetry/sdk-trace-node`](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-sdk-trace-node) package.
+
+Compatible with OpenTelemetry JS API and SDK `1.0+`.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/winston-transport
+```
+
+## Usage
+
+This package exports the Winston transport class that is used to send records to the
+OpenTelemetry Logs SDK. It can be used directly when configuring a Winston logger
```

Comment:
Doesn't the setup described here result in double instantiation of the transport? Once by user and once by the instrumentation?

Reply:
I agree, it would probably be good to link the docs of `@opentelemetry/instrumentation-winston` to let people know that they don't need to do both to avoid double instantiation.

Reply:
Added some extra details about @opentelemetry/instrumentation-winston